kbase-extension/static/kbase/js/widgets/narrative_core/kbaseNarrativeDataList.js

```diff
@@ -602,6 +602,9 @@ define([
                         if (obj.set_items) {
                             updateSetInfo(obj);
                         }
+                        const term = this.bsSearch.val();
+                        const type = this.$filterTypeSelect.find('option:selected').val();
+                        this.search(term, type);
```

Comment:
The only problem I see here is this is under the `objects.forEach` call, so `this.search` (and a panel re-render) will get called for each object in the loop.

I think if it's popped out, then this will be fine.

Reply:
Good catch! This is fixed in the latest commit.